### PR TITLE
Logs stdout from nightwatch task

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: lib/tasks/test.js [hunk 1]
```diff
@@ -177,10 +177,15 @@ module.exports.browserTest = function(gulp, config) {
...
 	var packageJsonContent = fs.readFileSync(path.join(process.cwd(), 'package.json'), 'utf8');
 
-	function cleanupTest() {
+	function cleanupTest(output) {
 		fs.unlink('Procfile');
 		fs.unlink(config.nightwatchConfig.replace('json', 'tmp.json'));
 		fs.writeFileSync(path.join(process.cwd(), 'package.json'), packageJsonContent, 'utf8');
+		
+		if (output && output.stdout) {
+			log.secondary(output.stdout);
```
Comment (samgiles) [inline lib/tasks/test.js]: Is stdout a stream?

Comment (AlbertoElias) [inline lib/tasks/test.js]: nope, it's the string returned from the command-line helper

**Comment by AlbertoElias** [comment]
Should now pass CI

**Comment by samgiles** [comment]
:+1:

**Pull request merged**